README.md

```diff
@@ -85,54 +141,16 @@ security of the system, such as auditing and explainability features.
 * Martin Vrachev ([@MVrachev](https://github.com/MVrachev)), VMware
 * Ricardo Aravena ([@raravena80](https://github.com/raravena80)), Rakuten
 * Lakshmi Manohar Velicheti ([@manohar9999](https://github.com/manohar9999)), Shape Security
-* JOIN OUR MEETINGS REGULARLY, THEN ADD YOURSELF VIA PULL REQUEST
-
-## Related Groups
-
-* [Kubernetes Policy Working Group](https://github.com/kubernetes/community/tree/master/wg-policy)
-* [Kubernetes SIG-Auth](https://github.com/kubernetes/community/tree/master/sig-auth)
-* [NIST Big Data WG](https://bigdatawg.nist.gov/)
-
-## History
+</details>
+JOIN OUR MEETINGS REGULARLY, THEN ADD YOURSELF VIA PULL REQUEST
 
-* SAFE WG - renamed to CNCF Security SIG
-* [(Proposed) CNCF Policy Working Group](/policy-wg-merging.md) - Merged into SAFE WG
 
 
-## Communications
+# Additional Information
 
-Anyone is welcome to join our open discussions of WG projects and share news related to the group's mission and charter. Much of the work of the group happens outside of WG meetings and we encourage project teams to share progress updates or post questions in these channels:
+## Meeting List
 
-* [Email list](https://lists.cncf.io/g/cncf-sig-security)
-* [CNCF Slack](https://slack.cncf.io/) #sig-security channel
-
-## Meeting Time
-
-The Security SIG group meets every Wednesday at 10:00am PT (USA Pacific):
-
-Join: https://zoom.us/my/cncfsigsecurity
-
-One tap mobile:
-* +16465588656,,7375677271# US (New York)
-* +16699006833,,7375677271# US (San Jose)
-
-Dial by your location:
-* +1 646 558 8656 US (New York)
-* +1 669 900 6833 US (San Jose)
-* 877 369 0926 US Toll-free
-* 855 880 1246 US Toll-free
-Meeting ID: 737 567 7271
-Find your local number: https://zoom.us/u/alwlmxlNn
-
-## In Person Meetings
-
-Please let us know if you are going and if you are interested in attending (or helping to organize!) an in-person meetup. Create a [github issue](https://github.com/cncf/sig-security/issues/new) for an event and add to list below:
-
-* KubeCon + CloudNativeCon, San Diego, CA - Nov 18 – 21, 2019 - [issue#128]
-
-[Past Events](past-events.md)
-
-## Meeting Minutes
+<details><summary>Click to view list</summary>
```

Comment:
non-blocking: I think this is better than it all in one long README... though was thinking that moving to a separate page would make it easier to link (same applies to next section)

Reply:
moved to `past-events.md`